File: GotadeOuroDefault.Web/Modules/Agendamento/Agendamento/AgendamentoForm.cs
using Serenity.ComponentModel;
using System;

namespace GotadeOuroDefault.Agendamento.Forms;

[FormScript("Agendamento.Agendamento")]
[BasedOnRow(typeof(AgendamentoRow), CheckNames = true)]
public class AgendamentoForm
{
    public int ClienteId { get; set; }
    public int VeiculoId { get; set; }
    public int ServicoId { get; set; }
    public DateTime DataAgendamento { get; set; }
    public TimeSpan HoraAgendamento { get; set; }
    public int IdFuncionario { get; set; }
}

File: GotadeOuroDefault.Web/Modules/ServerTypes/Agendamento/AgendamentoForm.ts
import { IntegerEditor, DateEditor, StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface AgendamentoForm {
    ClienteId: IntegerEditor;
    VeiculoId: IntegerEditor;
    ServicoId: IntegerEditor;
    DataAgendamento: DateEditor;
    HoraAgendamento: StringEditor;
    IdFuncionario: IntegerEditor;
}

export class AgendamentoForm extends PrefixedContext {
    static formKey = 'Agendamento.Agendamento';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!AgendamentoForm.init)  {
            AgendamentoForm.init = true;

            var w0 = IntegerEditor;
            var w1 = DateEditor;
            var w2 = StringEditor;

            initFormType(AgendamentoForm, [
                'ClienteId', w0,
                'VeiculoId', w0,
                'ServicoId', w0,
                'DataAgendamento', w1,
                'HoraAgendamento', w2,
                'IdFuncionario', w0
            ]);
        }
    }
}